includes/REST_API/Publisher_Logos_Controller.php

```diff
@@ -0,0 +1,435 @@
+<?php
+/**
+ * Class Publisher_Logos_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use Google\Web_Stories\Settings;
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Traits\Post_Type;
+use WP_Error;
+use WP_Post;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * Class to access publisher logos via the REST API.
+ *
+ * @since 1.12.0
+ */
+class Publisher_Logos_Controller extends REST_Controller {
+	use Post_Type;
+
+	/**
+	 * The namespace of this controller's route.
+	 *
+	 * @since 1.12.0
+	 * @var string
+	 */
+	protected $namespace = 'web-stories/v1';
+
+	/**
+	 * The base of this controller's route.
+	 *
+	 * @since 1.12.0
+	 * @var string
+	 */
+	protected $rest_base = 'publisher-logos';
```

Comment:
```suggestion
	/**
	 * Constructor.
	 */
	public function __construct() {
		$this->namespace = 'web-stories/v1';
		$this->rest_base = 'publisher-logos';
	}
```

Nitpic, but other controllers do this. I don't like the current implementation, even through it does the same thing, because it implies you are defining those properties and not setting them.

Reply:
I'm not sure I understand the explanation, but either approach works for me. It's indeed exactly the same thing.